Makefile

```diff
@@ -0,0 +1,6 @@
+build:
+	docker-compose build
+
+test: build
+	docker build -t nbviewer_test -f Dockerfile.tests .
+	docker run -e GITHUB_API_TOKEN nbviewer_test invoke test
```

Comment:
Does this successfully pass the token from the travis environment into the container?

Reply:
It passes any GITHUB_API_TOKEN that is in the outside environment into the container, yes. It does appear to work with the encrypted token set up currently on github.